pyproject.toml

```diff
@@ -1,12 +1,16 @@
 [build-system]
-requires = ["flit"]
-build-backend = "flit.buildapi"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
 
-[tool.flit.metadata]
-module = "fastapi"
-author = "Sebastián Ramírez"
-author-email = "tiangolo@gmail.com"
-home-page = "https://github.com/tiangolo/fastapi"
+[project]
+name = "fastapi"
+description = "FastAPI framework, high performance, easy to learn, fast to code, ready for production"
+readme = "README.md"
+requires-python = ">=3.7"
+license = "MIT"
```

Comment:
I get an error in the editor telling me that this should be a table, the spec seems to say the same: https://peps.python.org/pep-0621/#license

Is there any other spec that overrides that and allows using the string description?

Reply:
ye, I think it does.

see [here](https://github.com/samuelcolvin/dirty-equals/blob/5cf23ce233ec8804067bf8ac07272d16cd7cf1b9/pyproject.toml#L12) which as:
```toml
license = {file = 'LICENSE'}
```

Reply:
PEP 639 is already supported by Hatchling https://peps.python.org/pep-0639/

Reply:
Amazing, thanks!